apps/src/templates/instructions/TopInstructionsCSP.jsx

```diff
@@ -424,6 +433,16 @@ class TopInstructionsCSP extends Component {
                   teacherOnly={teacherOnly}
                 />
               )}
+              {this.props.viewAs === ViewType.Teacher &&
+                this.props.teacherMarkdown && (
```

Comment:
nit: it could clean this up a bit to extract `const displayTeacherFeedback`, which you could reference here and below